hack/kustomize-sub.sh

```diff
@@ -18,7 +18,7 @@ set -o nounset
 set -o pipefail
 
 root=$(dirname "${BASH_SOURCE[0]}")/..
-kustomize="${root}/hack/tools/bin/kustomize"
+source "${root}/hack/common_vars.sh"
 envsubst="${root}/hack/tools/bin/envsubst"
```

Comment:
Should `envsubst` (or `ENVSUBST`) go into `common-vars.sh` as well? @nawazkh WDYT?

Reply:
I concur, it would be consistent to have `ENVSUBST` also going into `common-vars.sh` and variables being capitalized (like `KUSTOMIIE` instead of `kustomize`).